01/userinfamous/wordvalue.py

```diff
@@ -0,0 +1,33 @@
+from data import LETTER_SCORES, DICTIONARY 
+import os
+
+#File path for Dictionary
+fname = os.path.join('../',DICTIONARY)
+
+#Wordlist from dictionary
+def load_words() -> list:
+    with open(fname,'r') as f:
+        wordlist = f.read().split()
+        return wordlist
+
+def calc_word_value(word:str) -> int:
+    score = 0
+    for letter in word:
+        if letter.upper() in LETTER_SCORES:
+            score += LETTER_SCORES[letter.upper()]
+    return score
+
+def max_word_value(words=None):
+    wordvalue_pair = list()
+    #checking if given argument
+    if words == None:
+        wordlist = load_words()
```

Comment:
you can use `wordlist = words if words else load_words()`. This is one line in comparison to four and it uses the fact that Python evaluates None expressions to False